README.md

```diff
@@ -58,7 +58,9 @@ Also you need to configure sass loader, since all the styles are in sass format.
 | isIndexColumnVisible      | boolean                 | false                                        | Is index column visible                     |
 | isItemBorderVisible       | boolean                 | true                                         | Is border visible between items             |
 | isSortable                | boolean                 | false                                        | Enable drag'n'drop sorting                  |
+| isShowOnlySelectedPassed  | boolean                 | false                                        | Is show only selected checkbox initial state passed |
```

Comment:
This could be more descriptive, for example "initialShowOnlySelectedValue" or "showOnlySelectedInitialValue"
Now it feels like this determines is value passed or not, not the value itself.

Reply:
Changed to showOnlySelectedInitialValue

Reply:
Changed to showOnlySelectedInitialValue